core/store/migrations/migrate.go

```diff
@@ -43,8 +44,13 @@ func Migrate(db *gorm.DB) error {
 			ID:      "1560881846",
 			Migrate: migration1560881846.Migrate,
 		},
+		{
+			ID:      "1560886530",
+			Migrate: migration1560886530.Migrate,
+		},
 	})
 
+	db.LogMode(true)
 	err := m.Migrate()
```

Comment:
Do we want to toggle it back to false after successful migration? Or do we want to keep it one for all future SQL work via ORM?

Reply:
It effectively gets toggled back by our config setup, see here: https://github.com/smartcontractkit/chainlink/pull/1396/files#diff-51b42c415a2523d9e9632bbde39ace6dR148